prefixing.graphql

```diff
@@ -0,0 +1,59 @@
+schema
+  @core(feature: "https://spec.example.com/featureA/v1.0")
+  @core(feature: "https://spec.example.com/featureB/v2.0", as: "B") {
+  query: Query
+}
+
+"""
+featureA_SomeType is a type defined by feature A.
```

Comment:
This document uses single underscores instead of double underscores in a bunch of places.

Reply:
EXAMPLE: This file still has a bunch of single underscores.